extensions/amp-story/1.0/amp-story.css

```diff
@@ -273,6 +273,22 @@ amp-story-grid-layer > * {
   pointer-events: auto !important;
 }
 
+.i-amphtml-story-grid-layer-z-index-reset amp-story-grid-layer,
+.i-amphtml-story-grid-layer-z-index-reset amp-story-grid-layer * {
+  contain: none !important;
+  z-index: auto !important;
+}
+
+amp-story-page.i-amphtml-story-grid-layer-z-index-reset[active] {
+  contain: none !important;
+  z-index: 100001 !important; /* On top of i-amphtml-story-system-layer. */
```

Comment:
We don't want the page to come on top of the system layer, instead, we want to hide the system layer.
You can toggle the `SYSTEM_UI_IS_VISIBLE_STATE` state, it will work right away.

Reply:
I was doing that before (hiding the system layer when an element is expanded), and I thought we preferred to place the overlay on top of the system layer.

I don't think I prefer either over the other, I'm just trying to understand why one is better than the other.

@newmuis do you have any opinions on this?

Reply:
Putting the whole grid-layer on top of the others can potentially affect other elements than the one you're trying to expand. My proposal was to: hide the system UI using that already existing action, and add a z-index only on the component you want to have on top of the rest.

Plus, we'd have to play with z-indexes every time we change/add any system components, and we know from experience that it can be really hard to maintain...